src/khoj/interface/web/config.html

```diff
@@ -398,7 +403,7 @@ <h3 class="card-title">
             }
 
             let notificationBanner = document.getElementById("notification-banner");
-            notificationBanner.innerHTML = "When updating the language model, be sure to delete all your saved content and re-initialize.";
+            notificationBanner.innerHTML = "Updated your embeddings model! When updating the language model, be sure to update all your saved content and re-initialize.";
```

Comment:
The use of *language model* and *embeddings model* maybe confusing. Maybe we can say something like:

```suggestion
            notificationBanner.innerHTML = "Khoj can now better understand the language of your content! Manually sync your data from one of the Khoj clients to have Khoj use it";
```